dist/palindrom-dom.js

```diff
@@ -2130,7 +2130,7 @@ var Palindrom = (function() {
     this.onLocalChange = options.onLocalChange || noop;
     this.onRemoteChange = options.onRemoteChange || noop;
     this.onStateReset = options.onStateReset || options.callback || noop;
-
+    this.filter = options.filter || (operation => operation);
```

Comment:
The name of this function suggests it acts like [Array.prototype.filter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter), which it does not. 

Maybe it would more consistent to call it `onBeforeLocalChange` or `filterLocalChange`.